UnityProject/Assets/Scripts/HealthV2/Living/GibbableMob.cs

```diff
@@ -19,17 +18,20 @@ public class GibbableMob : NetworkBehaviour, IGib
 		public PlayerScript Mob => mob;
 
 		[Server]
-		public void OnGib()
+		public void OnGib(bool ignoreNoGibRule = false)
 		{
 			PlayAudio();
-			if (IsEnabled == false || GameManager.Instance.GibbingAllowed == false)
+			if (ignoreNoGibRule == false)
 			{
-				Chat.AddActionMsgToChat(gameObject,
-					$"{mob.visibleName}'s body stretches violently before falling to the ground.".Color(Color.red));
-				mob.ObjectPhysics.NewtonianPush(new Vector2().RandomDirection(),500,
-					Random.Range(6, 55) / 100f, 8, (BodyPartType) Random.Range(0, 13), gameObject, Random.Range(0, 13));
-				health.Death();
-				return;
+				if (IsEnabled == false || GameManager.Instance.GibbingAllowed == false)
```

Comment:
what's the difference between these two?

Reply:
`IsEnabled` is per creature
`GibbingAllowed` is server wide

Reply:
I see